### ISSUE TITLE
Private variables in BSImagePickerViewController

### ISSUE THREAD

**Issue by newPrimitives** [mikaoj/BSImagePicker#53]
Hi, I think that private variables doneBarButton, cancelBarButton and albumTitleView in BSImagePickerViewController should be marked public. For example my app supports multiple languages, and I had to remove private prefix in order to set titles from my controller and next time i update my pods that change will be canceled. Also someone might prefer to have done button titled as upload or something similar. It's not a bug but i think it would be more useful to have them as public. Great library btw.

**Comment by mikaoj** [comment]
Hi!

Doesn't this work?
```swift
    // MARK: Buttons
    /**
    Cancel button
    */
    public var cancelButton: UIBarButtonItem {
        get {
            return self.cancelBarButton
        }
    }
    
    /**
    Done button
    */
    public var doneButton: UIBarButtonItem {
        get {
            return self.doneBarButton
        }
    }
    
    /**
    Album button
    */
    public var albumButton: UIButton {
        get {
            return self.albumTitleView.albumButton
        }
    }
```

**Comment by newPrimitives** [comment]
Nop, unless I'm doing something wrong.
Here's what I did: 
<code> 
     let vc = BSImagePickerViewController() 
</code>
<code> 
  vc.doneButton.title = "Something else"
</code>

I was able to modify other navbar properties such as color and background color without any problems though:

<code> 
   vc.navigationBar.setBackgroundImage(UIImage(named: "navigationRedColor"), forBarMetrics: .Default)
    vc.navigationBar.tintColor = UIColor.whiteColor()
</code>

**Comment by mikaoj** [comment]
Yeah, I don't think changing the title for a UIBarButtonSystemItem will have any effect. So you would have to init a new button. 
I'll take a look at making them public (probably inject them when initializing) whenever I get the time :)

**Comment by ilii** [comment]
Prepare a new version of the trial.

**Comment by mikaoj** [comment]
?

**Comment by mikaoj** [comment]
Fixed in 2.3.0
